pandas/core/base.py

```diff
@@ -943,18 +943,65 @@ def value_counts(self, normalize=False, sort=True, ascending=False,
             If True then the object returned will contain the relative
             frequencies of the unique values.
         sort : boolean, default True
-            Sort by values
+            Sort by values.
         ascending : boolean, default False
-            Sort in ascending order
+            Sort in ascending order.
         bins : integer, optional
             Rather than count values, group them into half-open bins,
-            a convenience for pd.cut, only works with numeric data
+            a convenience for pd.cut, only works with numeric data.
         dropna : boolean, default True
             Don't include counts of NaN.
 
         Returns
         -------
         counts : Series
+
+        Examples
+        --------
+
```

Comment:
I've noticed there's no blank line after section headers in https://python-sprints.github.io/pandas/guide/pandas_docstring.html

Reply:
Thanks, I have removed the unnecessary new line!